includes/class-sensei-course-participants-widget.php

```diff
@@ -180,8 +180,24 @@ public function widget( $args, $instance ) {
 					$learner_name = '<h3 itemprop="name" class="learner-name">' . esc_html( $display_name ) . '</h3>' . "\n";
 				}
 
+				$profile_url = false;
 				if ( true === $public_profiles ) {
 					$profile_url  = Sensei()->learner_profiles->get_permalink( $learner->ID );
+				}
+
+				/**
+				 * Filter the learner profile URL for course participants.
+				 *
+				 * @since 2.0.0
+				 *
+				 * @param string|bool $profile_url The learner's public profile URL if public profiles are enabled. Otherwise, this will be false.
+				 * @param WP_Post     $learner     The learner being displayed.
```

Comment:
Inspecting this, I think `$learner` is a `WP_User`

Reply:
Good catch! Fixed in https://github.com/woocommerce/sensei-course-participants/pull/36/commits/675a6740e6845b085ce31e3a58c4da728bcdcebc